bindings/gumjs/gumdukinterceptor.c

```diff
@@ -28,6 +28,7 @@ struct _GumDukInvocationContext
   GumInvocationContext * handle;
   GumDukHeapPtr cpu_context;
   gint depth;
+  GumDukInterceptor * interceptor;
```

Comment:
Let's put a blank line before this one, to be consistent with how we do it in `gumdukcore.c`.